### ISSUE TITLE
Guild rank and guild attack problems

### ISSUE THREAD

**Issue by SurvivorJochem** [Wynncraft/Issues#7437]
Hey guys,

Lately I wanted to give a guild member a higher rank but it didn't work, same as attacking a guild territory.
This showed up in chat when I wanted to attack the territory.

An internal error occured.

Or the chat said <guild> has taken over <territory> succesfull do /guild defend to defend this territory, and when I wanted to do that this showed up in the chat:

You must capture the territory first, or, An internal error occured.

Can this be solved soon?
Please notify me, Thank you.

**Comment by thecoolkogut4** [comment]
you can't promote him because most likely he/she changed their Mc name. can't attack because wars are 95% glitched. guild defend because it takes about a minute for the place to get updated that you own it (if you are reffering to the places in llivigar u can't defend them) 


guilds are really buggy right now, devs are working on a fix.

**Comment by ColinRioux** [comment]
We appreciate that you made an issue here, but I hope that in the future you use the search bar at the top of the issue page to search for current issues which are open. Changing a player's rank is a known issue/duplicate, and goes along with the name changing issue. In terms of "an internal error has occured", this is essentially another result of guild wars being broken, and the whole system needs to be written (and the developers are aware of it, yet I cannot comment on whether it's currently being worked on).

I also would like you to keep in mind that many other people besides yourself are having this issue, and all those who do have it hopes that this will be fixed as soon as possible (or people have lost all hope). 

tl:dr Cannot comment if this issue is on the list of being fixed (as I don't know), I cannot guarantee it'll be fixed when you wake up in the morning or if it'll be fixed in a few months. The whole guild system is broken.
